nvtabular/workflow/workflow.py

```diff
@@ -460,6 +476,19 @@ def _transform_partition(root_df, workflow_nodes, additional_columns=None):
                 # use input_columns to ensure correct grouping (subgroups)
                 selection = node.input_columns.resolve(node.input_schema)
                 output_df = node.op.transform(selection, input_df)
+
+                # Update or validate output_df dtypes
+                for col_name, col_schema in node.output_schema.column_schemas.items():
+                    output_schema = col_schema.with_dtype(output_df[col_name].dtype)
+
+                    if capture_dtypes:
+                        node.output_schema.column_schemas[col_name] = output_schema
+                #     else:
```

Comment:
We should still try this, but it can probably be removed from this PR for the time being.